content/news/2025-09-24-bevy-0.17/index.md

```diff
@@ -60,13 +60,16 @@ Additionally, if you have a NVIDIA GPU, you can enable DLSS Ray Reconstruction w
 
 ### How it works
 
-TODO: Showcase the different aspects/steps of Solari (direct initial+temporal+spatial, indirect initial+temporal+spatial, world cache, DLSS RR)
+Our current implementation uses rayraced dissue direct and indirect lighting (also known as global illuminance), which are sampled via ReSTIR variants and uses a world-space irradiance cache to improve quality.
```

Comment:
> rayraced dissue

This doesn't sound like a thing to me!

Reply:
I have no idea what went wrong there 😅 Do review this bit carefully though: I'm in over my head even trying to summarize.